src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const findedTab = tabs.find(tab => tab.id === activeTabId) ?? tabs[0];
```

Comment:
AI is right its bad naming
```suggestion
  const activeTab = tabs.find(tab => tab.id === activeTabId) ?? tabs[0];
```